dashboard/test/ui/README.md

```diff
@@ -71,23 +71,23 @@ Run all UI tests for a given browser
 
```

Comment:
These changes reflect that features are now consistently named with snake_case rather than camelCase (#20938), because it is more in line with Ruby conventions and Cucumber tests are Ruby-ish.